people.json

```diff
@@ -0,0 +1 @@
+[]
```

Comment:
- [ ] After adding a person, the following error appears when trying to show the list of people. Kindly fix it.

![image](https://github.com/Zel-hub7/school-library/assets/62290545/15b60ad0-0e5c-4cc2-99cb-b55bce99dc67)